spec/factories/kudos.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :kudo do
+    title { Faker::Superhero.name }
+    content { Faker::TvShows::Friends.quote }
+    giver_id { Faker::Number.between(from: 1, to: 5) }
+    receiver_id { Faker::Number.between(from: 1, to: 5) }
```

Comment:
Entering hard-coded values as identifiers for related objects can be problematic.
Now if you do not create 5 employees there will be error.

Factory Bot offers solution for that - it can create all associated objects automatically:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

Better will be:
```suggestion
    giver factory: :employee
    receiver factory: :employee
```

Reply:
Looks better. Thanks for advice!